schema-loader/src/integration-test/java/com/scalar/db/schemaloader/SchemaLoaderIntegrationTestBase.java

```diff
@@ -0,0 +1,179 @@
+        SCHEMA_FILE,
+        "--coordinator");
+  }
+
+  protected List<String> getSchemaLoaderDeletionCommandArgs() {
+    List<String> args = new ArrayList<>(getSchemaLoaderCreationCommandArgs());
+    args.add("-D");
+    return args;
+  }
+
+  @Test
+  public void storageSpecificCommand_GivenProperArgs_ShouldCreateTablesThenDeleteTables()
+      throws Exception {
+    createTables_GivenProperSchemaFileAndStorageSpecificCommandArgs_ShouldCreateTables();
+    deleteTables_GivenProperSchemaFileAndStorageSpecificCommandArgs_ShouldDeleteTables();
+  }
+
+  @Test
+  public void schemaLoaderCommand_GivenProperArgs_ShouldCreateTablesThenDeleteTables()
+      throws Exception {
+    createTables_GivenProperSchemaFileAndSchemaLoaderCommandArgs_ShouldCreateTables();
+    deleteTables_GivenProperSchemaFileAndSchemaLoaderCommandArgs_ShouldDeleteTables();
+  }
+
+  public void createTables_GivenProperSchemaFileAndStorageSpecificCommandArgs_ShouldCreateTables()
+      throws Exception {
+    createTables_GivenProperSchemaFileAndCommandArgs_ShouldCreateTables(
+        getStorageSpecificCreationCommandArgs());
+  }
+
+  public void createTables_GivenProperSchemaFileAndSchemaLoaderCommandArgs_ShouldCreateTables()
+      throws Exception {
+    createTables_GivenProperSchemaFileAndCommandArgs_ShouldCreateTables(
+        getSchemaLoaderCreationCommandArgs());
+  }
+
+  private void createTables_GivenProperSchemaFileAndCommandArgs_ShouldCreateTables(
+      List<String> args) throws Exception {
+    // Act
+    int exitCode = executeCommandWithArgs(args);
+
+    // Assert
+    assertTableCreation(exitCode);
+  }
+
+  public void deleteTables_GivenProperSchemaFileAndStorageSpecificCommandArgs_ShouldDeleteTables()
+      throws Exception {
+    deleteTables_GivenProperSchemaFileAndCommandArgs_ShouldDeleteTables(
+        getStorageSpecificDeletionCommandArgs());
+  }
+
+  public void deleteTables_GivenProperSchemaFileAndSchemaLoaderCommandArgs_ShouldDeleteTables()
+      throws Exception {
+    deleteTables_GivenProperSchemaFileAndCommandArgs_ShouldDeleteTables(
+        getSchemaLoaderDeletionCommandArgs());
+  }
+
+  private void deleteTables_GivenProperSchemaFileAndCommandArgs_ShouldDeleteTables(
+      List<String> args) throws Exception {
+    // Act
+    int exitCode = executeCommandWithArgs(args);
+
+    // Assert
+    assertTableDeletion(exitCode);
+  }
+
+  private int executeCommandWithArgs(List<String> args) throws Exception {
+    // Arrange
+    ProcessBuilder processBuilder = new ProcessBuilder(args);
+
+    // Act
+    Process process = processBuilder.start();
+    try {
+      try (final BufferedReader input =
+          new BufferedReader(
+              new InputStreamReader(process.getErrorStream(), Charset.defaultCharset()))) {
+        String line;
+        while ((line = input.readLine()) != null) {
+          System.out.println(line);
+        }
+      }
+    } catch (Exception e) {
+      System.err.println(e.toString());
```

Comment:
We should print stack trace:
```suggestion
      e.printStackTrace(System.err);
```

Reply:
Updated in [commit](https://github.com/scalar-labs/scalardb/pull/349/commits/a092f40750397b43dba8e63a10aa802d77b36050)

Reply:
Updated in [commit](https://github.com/scalar-labs/scalardb/pull/349/commits/ffc130bde7d8d47ce6d2e9273fe6eef7c5f4bd2d)